patches/components-permissions-android-permission_prompt_android.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/permissions/android/permission_prompt_android.cc b/components/permissions/android/permission_prompt_android.cc
+index 0486024f3751eb844b3ad677447490aa7622b023..42a135d625a4335369a5c8b0ba70c1edb91e3e3f 100644
+--- a/components/permissions/android/permission_prompt_android.cc
++++ b/components/permissions/android/permission_prompt_android.cc
+@@ -144,6 +144,7 @@ static bool IsValidARCameraAccessRequestGroup(
+ // AR + Camera, or Camera + AR.
+ static void CheckValidRequestGroup(
+     const std::vector<permissions::PermissionRequest*>& requests) {
++  if (requests.size() > 0 && requests[0]->request_type() == permissions::RequestType::kBraveEthereum) return;
```

Comment:
these patches don't follow patching standards and we seem to be doing something different here in terms of grouped permissions than we did on desktop cc @yrliou

Reply:
also this would allow ethereum permissions to be grouped with anything and they should only be grouped with other ethereum permissions

Reply:
that code is removed and we have a permission dialog in a browser process now